index.html

```diff
@@ -0,0 +1,148 @@
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/fork-awesome@1.2.0/css/fork-awesome.min.css"
+      integrity="sha256-XoaMnoYC5TH6/+ihMEnospgm0J1PM/nioxbOUdnM8HY="
+      crossorigin="anonymous"
+    />
+    <title>ANDIS</title>
+  </head>
+  <body>
+    <header>
+      <div id="secondary-nav-bar" class="container">
+        <div class="wrapper">
+          <a href="#"><i class="fa fa-mastodon"></i></a>
+          <a href="#"><i class="fa fa-twitter"></i></a>
+        </div>
+      </div>
+      <div id="nav-bar">
+        <a href="#" id="logo">ANDIS<span>2022</span></a>
+        <nav id="menu">
+          <span id="menu-close-btn">×</span>
+          <a href="#" class="home-link">Home</a>
+          <a href="#program">Program</a>
+          <a href="#speakers">Speakers</a>
+          <a href="#sponsors">Sponsors</a>
+          <a href="./about.html">About</a>
+        </nav>
+        <a href="#" id="menu-toggler">
+          <svg
+            class="w-6 h-6"
+            fill="none"
+            stroke="currentColor"
+            viewBox="0 0 24 24"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path stroke-width="3" d="M4 6h16M4 12h16M4 18h16"></path>
+          </svg>
+        </a>
+      </div>
+    </header>
+
+    <section id="section1" class="container">
+      <div class="wrapper">
+        <img src="./assets/img/shape-set-01.png" class="shape-set" alt="" />
+        <h1>Anambra State Town Hall Meeting 2022</h1>
+        <h2>"Sustainable Development"</h2>
+        <p class="description">
+          A multi-sectoral model perdicated on the internationally acclaimed
+          Millennium Development Goals [MDGs] of Eradication of Extreme Poverty
+          & Hunger, Gender Equality & Women Empowerment.
+        </p>
+        <h3>24th June, 2022 (Friday)</h3>
+        <p class="location">St Luke's Scondary School, Awka</p>
+      </div>
+    </section>
+
+    <section id="program">
+      <h2 class="section-heading">Main Program</h2>
+      <div class="items">
+        <div class="item">
+          <img src="./assets/img/icons-1.svg" class="item-icon" alt="" />
+          <h3 class="item-title">Lecture</h3>
+          <p class="item-description">
+            Lecture on the importance of Town Hall meetings by the Minister of
+            Education
+          </p>
+        </div>
+        <div class="item">
+          <img src="./assets/img/icons-2.svg" class="item-icon" alt="" />
+          <h3 class="item-title">Interaction</h3>
+          <p class="item-description">
+            Question and Answer session with house members and leaders of the
+            state
+          </p>
+        </div>
+        <div class="item">
+          <img src="./assets/img/icons-3.svg" class="item-icon" alt="" />
+          <h3 class="item-title">Awards</h3>
+          <p class="item-description">
+            Awards for Civil Servants for outstanding service
+          </p>
+        </div>
+      </div>
```

Comment:
- [ ] Your design is really good kindly just add two more to match the layout of the original design.

![image](https://user-images.githubusercontent.com/35535130/169192255-46205a6f-4d5f-4b98-b293-01a3ef9da46d.png)

Reply:
Kindly make this request change from the previous code review. Follow design guide lines